docs/root/configuration/http/http_filters/admission_control_filter.rst

```diff
@@ -77,38 +80,11 @@ Example Configuration
 An example filter configuration can be found below. Not all fields are required and many of the
 fields can be overridden via runtime settings.
 
-.. code-block:: yaml
-
-  name: envoy.filters.http.admission_control
-  typed_config:
-    "@type": type.googleapis.com/envoy.extensions.filters.http.admission_control.v3.AdmissionControl
-    enabled:
-      default_value: true
-      runtime_key: "admission_control.enabled"
-    sampling_window: 120s
-    sr_threshold:
-      default_value: 95.0
-      runtime_key: "admission_control.sr_threshold"
-    aggression:
-      default_value: 1.5
-      runtime_key: "admission_control.aggression"
-    rps_threshold:
-      default_value: 5
-      runtime_key: "admission_control.rps_threshold"
-    max_rejection_probability:
-      default_value: 80.0
-      runtime_key: "admission_control.max_rejection_probability"
-    success_criteria:
-      http_criteria:
-        http_success_status:
-          - start: 100
-            end:   400
-          - start: 404
-            end:   404
-      grpc_criteria:
-        grpc_success_status:
-          - 0
-          - 1
+.. literalinclude:: _include/admission-control-filter.yaml
+    :language: yaml
```

Comment:
lets add `:linenos:` - we havent been so consistent with how we present these, but i think its helpful to have line numbers